api/v1beta1/awscluster_conversion.go

```diff
@@ -17,40 +17,41 @@ limitations under the License.
 package v1beta1
 
 import (
-	infrav1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+	infrav2 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
```

Comment:
This is wrong sadly. 🤣 I'll include a sample of how this needs to look like. I know the conversion stuff is pretty complex so it's hard to follow what to do with this thing.

Reply:
I can't read for shit. I thought this renamed the import 🤦 This is actually correct.